Mathlib/Tactic/Linter/Lint.lean

```diff
@@ -354,6 +354,7 @@ def longFileLinter : Linter where run := withSetOptionIn fun stx ↦ do
   if let some init := stx.getTailPos? then
     -- the last line: we subtract 1, since the last line is expected to be empty
     let lastLine := ((← getFileMap).toPosition init).line
+    -- in this case, the file has an allowed length, and the linter option is unnecessarily set
     if lastLine ≤ defValue && defValue < linterBound then
```

Comment:
Can you check that some test fails if you remove the second condition in the `if`? If not, please add a test :-)

Reply:
Thanks!